config/was.yaml

```diff
@@ -189,15 +197,40 @@ output:
             realized_flux: "@realized_flux"
             flux: "@flux"
             mag: "@mag"
+            mag_AB: { type: Eval, str: '-2.5 * np.log10(@flux/@image.exptime/euclidlike.collecting_area) + euclid_AB_ZP' }
             obj_type: "@object_type"
-            gal_disk_hlr: { type: SkyCatValue, field: diskHalfLightRadiusArcsec }
             gal_redshift: { type: SkyCatValue, field: redshift }
+            gal_disk_hlr: { type: SkyCatValue, field: diskHalfLightRadiusArcsec }
+            gal_disk_e1: { type: SkyCatValue, field: diskEllipticity1 }
+            gal_disk_e2: { type: SkyCatValue, field: diskEllipticity2 }
+            gal_bulge_hlr: { type: SkyCatValue, field: spheroidHalfLightRadiusArcsec }
+            gal_bulge_e1: { type: SkyCatValue, field: spheroidEllipticity1 }
+            gal_bulge_e2: { type: SkyCatValue, field: spheroidEllipticity2 }
+            gal_shear1: { type: SkyCatValue, field: shear1 }
+            gal_shear2: { type: SkyCatValue, field: shear2 }
+            gal_convergence: { type: SkyCatValue, field: convergence }
             sn_host_id: { type: SkyCatValue, field: host_id }
-            lsst_flux_r:
+            lsst_flux_u: { type: SkyCatValue, field: lsst_flux_u }
+            lsst_mag_u:
+                type : Eval
+                str : "-2.5 * np.log10(@output.truth.columns.lsst_flux_u) + 12.628"
```

Comment:
Should document in a comment the source for the magic zeropoint numbers in the LSST bands.